webapp-django/crashstats/crashstats/views.py

```diff
@@ -31,6 +31,17 @@
 datetime.datetime.strptime('2013-07-15 10:00:00', '%Y-%m-%d %H:%M:%S')
 
 
+def ratelimit_blocked(request, exception):
+    # http://tools.ietf.org/html/rfc6585#page-3
+    status = 429
+    if request.path.startswith('/api/'):
+        # return a super spartan message
+        # we could also do something like `{"error": "Too Many Requests"}`
```

Comment:
I've been wondering about this when building APIs recently and I think this is unnecessary. It's true that a string is not a valid JSON output, but do API consumers bother to parse the content when the status of the request is not a 200? I personally don't, so I would not bother making the output a JSON content.